content/en/docs/community/release_checklist.md

```diff
@@ -126,31 +159,34 @@ git tag --sign --annotate "$RELEASE_NAME" --message "Helm release $RELEASE_NAME"
 git push upstream "$RELEASE_NAME"
 ```
 
-This new tag is going to be the base for the patch release.
+This new tag is going to be the base for the patch release. Creating a
+[helm/helm milestone](https://github.com/helm/helm/milestones) is optional for
+patch releases.
 
 Make sure to check [helm on CircleCI](https://circleci.com/gh/helm/helm) to see
-that the release passed CI before proceeding.
+that the release passed CI before proceeding. Patch releases can skip steps 2-5
+and proceed to step 6 to [Finalize the Release](#6-finalize-the-release).
```

Comment:
Should this be proceed to step 7?

Reply:
According to @bacongobbler it's necessary to finalize patch releases as shown in step 6.

Reply:
Hmm, in re-reading it does seem like those steps are duplicated. Let me try to clear that up.